plugins/scaffolder-actions/scaffolder-backend-module-aws/src/actions/s3/cp.test.ts

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright 2021 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { getVoidLogger } from '@backstage/backend-common';
+import { createAwsS3CpAction } from './cp';
+import { PassThrough } from 'stream';
+import mockFs from 'mock-fs';
+
+const mockS3Client = {
```

Comment:
There is a library called aws-sdk-client-mock that I think will be easier to work with.

Reply:
Yes, I tried to make it work, however I was not able to There was an ugly `unable to find module something something error` Let me repor again and post it here why I didnt use that. I think it's something with the transpile config or idk. So I opted for now to manually mock it. I suggest we try to use the aws sdk client mock when we add more actions here and it'll need in more packages